packages/e2e-test-utils-playwright/src/editor/insert-block.ts

```diff
@@ -41,4 +41,5 @@ async function insertBlock(
 	}, blockRepresentation );
 }
 
+export type { BlockRepresentation };
```

Comment:
Without this, TypeScript will throw an error like `Public property 'insertBlock' of exported class has or is using name 'BlockRepresentation' from external module "/Users/kai/work/gutenberg/packages/e2e-test-utils-playwright/src/editor/insert-block" but cannot be named.`